### PR TITLE
Create an endpoint to list tokens for given user.

### PR DESCRIPTION
**Pull request by Buwujiu**

### CODE REVIEW

File: src/main/java/bio/overture/ego/controller/TokenController.java [hunk 1]
```diff
@@ -98,9 +98,12 @@ public TokenController(@NonNull TokenService tokenService) {
     return format("Token '%s' is successfully revoked!", token);
   }
 
-  @ResponseBody
-  List<TokenResponse> listTokens(@RequestHeader(value = "Authorization") String authorization) {
-    return null;
+  @RequestMapping(method = RequestMethod.GET, value = "/list_token")
```
Comment (andricDu) [inline src/main/java/bio/overture/ego/controller/TokenController.java]: lets rename this to just token.

This way we get proper REST usaged with GET, POST, and DELTE for /token.

I realize now the check_token endpoint must be confusing you. The reason that exists is for backwards compatibility with ICGC.

Comment (andricDu) [inline src/main/java/bio/overture/ego/controller/TokenController.java]: I think you are missing the annotation `@AdminScoped` to make sure only an admin user or application can list this.

**Comment by Buwujiu** [comment]
ticket #245

**Comment by andricDu** [comment]
Thanks!

**Pull request merged**